src/Snake.DataAccess.IntegrationTests/UsersTests.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.EntityFrameworkCore;
+using Snake.Game.Models;
+using Xunit;
+
+namespace Snake.DataAccess.IntegrationTests
+{
+    public class UsersTests
+    {
+        private SnakeDBContext CreateContext()
+        {
+            var builder = new DbContextOptionsBuilder<SnakeDBContext>().UseInMemoryDatabase();
+            return new SnakeDBContext(builder.Options);
+        }
+
+        [Fact]
+        public void Add_NewUser_ShouldExist()
+        {
+            // Arrange
+            var context = CreateContext();
+
+            const int userId = 1;
+            var user = new User
+            {
+                UserId = userId,
+                Nickname = "johny",
+                FullName = "Johny Johnson"
+            };
+
+            // Act
+            context.Users.Add(user);
+            var expected = context.Users.Find(userId);
```

Comment:
You're testing the EntityFramework context. You shouldn't test it. This test is not irrelevant.